src/extension/app/components/modal/modal-container.js

```diff
@@ -162,19 +162,10 @@ export class ModalContainer extends LitElement {
     const { type, data } = modal;
     const options = {};
     switch (type) {
-      case MODALS.WAIT:
-        options.dismissable = false;
-        options.content = html`
-          <div class="wait-dialog">
-            <sp-progress-circle label=${data.message} indeterminate></sp-progress-circle>
-            <span>${data.message}</span>
-          </div>
-        `;
-        break;
       case MODALS.ERROR:
```

Comment:
No longer using the Error modal as it's all been moved into toasts in the action-bar. Figure we should still keep this in case it's needed in the future.